dataset_config/platinum_genomes/gcs.json

```diff
@@ -0,0 +1,13 @@
+// Fields for GCS indexer
+{
+  "gcs_paths": [
```

Comment:
Convention is to name this `gcs_pattern` indicating a regex string over `gcs_path` which indicates a static file path. I would also explicitly mention that regex is allowed here in addition to treating PRIMARY_KEY as a special substring.

I would prefer that users not have to supply regex, but this seems to be the easiest workaround for the time being.